### DISCUSSION TITLE
Consuming stacktraces

### DISCUSSION THREAD

**Discussion by vheon** [stevearc/overseer.nvim#176] [Q&A] [answered]
Looking around the documentation I read this https://github.com/stevearc/overseer.nvim/blob/master/doc/parsers.md#parsing-a-golang-stack-trace which seems really cool but I still don't understand the intended way to consume such information 😕 Is that thought as an integration with a specific plugin? I was thinking it would be useful for errors of C++ template instantiation where you  have a similar structure but it is not clear to me how to consume the extracted information then.

**Comment by stevearc** [comment, marked as answer]
The origins of this are that I initially conceived of overseer as a code + test runner, and I wanted to be able to parse stack traces of failures and set them into the quickfix or do other things with them. However, [neotest](https://github.com/nvim-neotest/neotest) was released while I was still working on the first version, so I scrapped the testing features and focused on the code running instead. However, I kept some of the tools that I built for the testing (that's where the parsers came from), and I decided to keep the example of parsing a Go stack trace in the docs because I thought it was a good showcase of how to use parsers.

All that is to say that at the moment there are no features within overseer to do anything special with stack traces. If you have some ideas for what you would like to do with it, I'd be happy to help you figure out how to make that happen!

**Reply by vheon** [reply]
Thanks! Makes perfect sense as a showcase and I'll ping you for sure if I need some help around it 👍
